content/documentation/3-stream-developer-guides/2-streams/1-standalone-stream-rabbitmq.md

```diff
@@ -0,0 +1,229 @@
+---
+path: 'stream-developer-guides/streams/standalone-stream/'
+title: 'Standalone Streaming Application Development on RabbitMQ'
+description: 'Create a simple stream processing application on RabbitMQ'
+---
+
+# Stream Processing with Apache Kafka
+
+In this guide we will develop a stream processing application and deploy it to Cloud Foundry, Kubernetes, and on your local machine.
+In another guide, we will deploy the stream processing application using Data Flow.
+
+We will start from initializr and create two Spring Cloud Stream applications.
+Note for CF we need a manifest, for k8s we need a service/deployment yaml.
+
+## Development
+
+### Initialzr
+
+Go to [Spring Cloud Stream Initializr](https://start-scs.cfapps.io/) and
+
+- Generate a `Maven Project` With Spring Boot `2.1.4` (Choose the version of your choice)
+- Select Group `com.example`
+- Select Artifact `demo-source`
+- Search for `RabbitMQ binder starter` dependency
+- Click `Generate Project`
+
```

Comment:
Include a link so that the user can click a preconfigured zip file .  Shown here in the batch PR: https://github.com/spring-io/dataflow.spring.io/pull/6/files#diff-dc94a800788a54f1beaeb53b17fd8e31R43